src/gui/qgsnewvectorlayerdialog.h

```diff
@@ -44,6 +44,11 @@ class GUI_EXPORT QgsNewVectorLayerDialog: public QDialog, private Ui::QgsNewVect
     static QString runAndCreateLayer( QWidget *parent = nullptr, QString *enc = nullptr, const QgsCoordinateReferenceSystem &crs = QgsCoordinateReferenceSystem(),
                                       const QString &initialPath = QString() );
 
+    /**
+     * New dialog constructor.
+     * \param parent parent object
```

Comment:
Better remove those params again. Doxygen is quite picky once you start documenting params like this. Mostly inline docs with `\a parent is used to specify the parent object` but in this case here they are better skipped (because current docs are just stating the obvious and the param name itself is as informative as the description)

Reply:
Just did that change. Thanks for the suggestion.